pkg/netstat/route.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package netstat
+
+import (
+	"strings"
+)
+
+const (
+	RTFUP        uint32 = 0x0001
+	RTFGATEWAY   uint32 = 0x0002
+	RTFHOST      uint32 = 0x0004
+	RTFREINSTATE uint32 = 0x0008
+	RTFDYNAMIC   uint32 = 0x0010
+	RTFMODIFIED  uint32 = 0x0020
+	RTFMTU       uint32 = 0x0040
+	RTFMSS       uint32 = RTFMTU
+	RTFWINDOW    uint32 = 0x0080
+	RTFIRTT      uint32 = 0x0100
+	RTFREJECT    uint32 = 0x0200
+	RTFNOTCACHED uint32 = 0x0400
+
+	RTFDEFAULT   uint32 = 0x00010000
+	RTFALLONLINK uint32 = 0x00020000
+	RTFADDRCONF  uint32 = 0x00040000
+	RTFNONEXTHOP uint32 = 0x00200000
+	RTFEXPIRES   uint32 = 0x00400000
+	RTFCACHE     uint32 = 0x01000000
+	RTFFLOW      uint32 = 0x02000000
+	RTFPOLICY    uint32 = 0x04000000
+	RTFLOCAL     uint32 = 0x80000000
+)
+
+func convertFlagData(flag uint32) string {
+	var s strings.Builder
+	if flag&RTFUP > 0 {
```

Comment:
I have to wonder: could this be an array: 
type flags []struct {
flag uint32 
n string}{
RFGATEWAY, "G"
...
}
for _, f := range flags {
if flag& f.flag {
s.WriteString(f.n)
}

Reply:
Thanks for the suggestion. Much better way to do things. Will put that in my toolbox. 

Need to finish most of the other flags first.